doc/RPC.md

```diff
@@ -0,0 +1,1356 @@
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "value": 50
+  }
+}
+```
+
+### get_node_status
+
+Returns current taraxa network protocol version
+
+#### Parameters
+
+none
+
+#### Returns
+
+`OBJECT` - with a lot of info about node status including networking. See Example
+
+#### Example 
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"get_node_status","params":[],"id":1}'
+
+// Result
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "blk_count": 6033325,
+    "blk_executed": 6033312,
+    "blk_queue_size": 0,
+    "dag_level": 3058195,
+    "dpos_node_votes": 0,
+    "dpos_quorum": 667,
+    "dpos_total_votes": 15300,
+    "network": {
+      "peer_max_node_dag_level": 3058195,
+      "peer_max_node_dag_level_node_id": "26d873a8fe9f3fe763491bfb1e0fcb3e95c7f5ca0d791f22936f62ba1ece9f0384c347f0ed642f997fe93203781af8116180458f66ed27fa58e93cd7fddc8947",
+      "peer_max_pbft_chain_size": 324432,
+      "peer_max_pbft_chain_size_node_id": "dd607f2ec9daa9f47335d0da859ebc2d1b489085c8b918f5ca3f13c0c8e6a29b731be4adf0c9c729baf932a656bcc4b6c9e2b10e9aa906b170f2c271504ababc",
+      "peer_max_pbft_round": 324617,
+      "peer_max_pbft_round_node_id": "26d873a8fe9f3fe763491bfb1e0fcb3e95c7f5ca0d791f22936f62ba1ece9f0384c347f0ed642f997fe93203781af8116180458f66ed27fa58e93cd7fddc8947",
+      "peers": [
+        {
+          "dag_level": 3058195,
+          "dag_synced": true,
+          "node_id": "26d873a8fe9f3fe763491bfb1e0fcb3e95c7f5ca0d791f22936f62ba1ece9f0384c347f0ed642f997fe93203781af8116180458f66ed27fa58e93cd7fddc8947",
+          "pbft_size": 324431
+        }, ...
+      ]
+    },
+    "node_count": 79,
+    "pbft_round": 324617,
+    "pbft_size": 324432,
+    "pbft_sync_period": 324432,
+    "pbft_sync_queue_size": 0,
+    "peer_count": 50,
+    "synced": true,
+    "syncing_seconds": 0,
+    "trx_count": 28017090,
+    "trx_executed": 28017007,
+    "trx_nonfinalized_size": 83,
+    "trx_pool_size": 7
+  }
+}
+```
+
+### get_packets_stats
+
+Returns current taraxa network protocol version
+
+#### Parameters
+
+none
+
+#### Returns
+
+`OBJECT` - with info about packets statistic. See example
+
+#### Example 
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"get_packets_stats","params":[],"id":1}'
+
+// Result
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "received_packets": [
+      {
+        "avg_processing_duration": 363,
+        "avg_size": 5,
+        "avg_tp_wait_duration": 543,
+        "total_count": 243,
+        "total_processing_duration": 88363,
+        "total_size": 1215,
+        "total_tp_wait_duration": 132164,
+        "type": "GetPbftSyncPacket"
+      },
+      {
+        "avg_processing_duration": 1804,
+        "avg_size": 488,
+        "avg_tp_wait_duration": 2270,
+        "total_count": 2145,
+        "total_processing_duration": 3870713,
+        "total_size": 1047430,
+        "total_tp_wait_duration": 4869225,
+        "type": "VotesSyncPacket"
+      },
```

Comment:
There is a lot of spam in this response example, we could put here only 1 object:
```
{
        "avg_processing_duration": 363,
        "avg_size": 5,
        "avg_tp_wait_duration": 543,
        "total_count": 243,
        "total_processing_duration": 88363,
        "total_size": 1215,
        "total_tp_wait_duration": 132164,
        "type": "GetPbftSyncPacket"
      }
```

and then write somewhere list of all possible values for "type"